src/com/dimagi/test/external/ExternalAppActivity.java

```diff
@@ -117,9 +120,34 @@ protected void onCreate(Bundle savedInstanceState) {
             ExternalAppActivity.this.startActivity(i);
         });
 
+        Button btnCheckUserSessionStatus = this.findViewById(R.id.button_test_user_session_status);
+        btnCheckUserSessionStatus.setOnClickListener(v -> {
+            if (isUserSessionActive()) {
```

Comment:
Nit: The if-else only needs to be on the text to show, then showing the toast for the chosen text could happen after the branch.